README.md

```diff
@@ -1,46 +1,81 @@
 
-<h1 align="center">
-  <br>
-  VS Code - Microsoft Edge Tools
-  <br>
-</h1>
+# Microsoft Edge Developer Tools for Visual Studio Code
 
-<h4 align="center">Show the browser's Elements and Network tool inside the VS Code editor and use it to fix styling, layout, and CSS issues with your site.</h4>
+**Show the browser's Elements and Network tool inside the Visual Studio Code editor and use it to fix CSS issues with your site and inspect the Network.**
 
-A VS Code extension that allows you to use the browser's Elements and Network tool from within the editor. The DevTools will connect to an instance of Microsoft Edge giving you the ability to see the runtime HTML structure, alter layout, fix styling issues, and view network requests. All without leaving VS Code.
+A Visual Studio Code extension that allows you to use the browser's Elements and Network tool from within the editor. The DevTools will connect to an instance of Microsoft Edge giving you the ability to see the runtime HTML structure, alter layout, fix styling issues, and view network requests. All without leaving Visual Studio Code.
 
-**Note**: This extension only supports Microsoft Edge
+**Note**: This extension _only_ supports Microsoft Edge
 
-![Microsoft Edge Tools - Demo](demo.gif)
+![Microsoft Edge Tools - Demo](basic_usage.gif)
```

Comment:
Do we also want to show network tool functionality here or is the main focus of the extension the Elements panel?

Reply:
That's only the intro one. As Network isn't enabled by default, this would be confusing.